sematic/api/endpoints/resolutions.py

```diff
@@ -53,16 +57,8 @@ def get_resolution_endpoint(user: Optional[User], resolution_id: str) -> flask.R
             HTTPStatus.NOT_FOUND,
         )
 
-    resolution_json = resolution.to_json_encodable()
-
-    # Scrub the environment variables before returning from the
-    # API. They can contain sensitive info like API keys. On write,
-    # we consider this field to be immutable, so we will just re-use
-    # whatever was already in the DB for it
-    resolution_json[Resolution.settings_env_vars.key] = {}
```

Comment:
We already have a redaction mechanism that is closer to the models.

Reply:
We can simply skip absent keys in `JSONEncodableMixin.from_json_encodable` and the regular default factory will handle it.